### PR TITLE
Private registry guide 

### PR DESCRIPTION
**Pull request by jonathanschwarze**
https://github.com/sapcc/kubernikus/issues/164

### CODE REVIEW

File: docs/guide/common_addons.md [hunk 1]
```diff
@@ -71,4 +71,155 @@ dashboard accessible from the outside via `https` on port `443`.
...
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: registry-storage
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
```
Comment (SchwarzM) [inline docs/guide/common_addons.md]: 1g seems too small.

File: docs/guide/common_addons.md [hunk 2]
```diff
@@ -71,4 +71,155 @@ dashboard accessible from the outside via `https` on port `443`.
...
+        ports:
+        - name: registry
+          containerPort: 80
+          hostPort: 5000
+```
+
+### Access registry from outside
+Through a ssh tunnel you can push or pull images from your cluster registry. At first, export your local ip: 
+```
+export local_ip=$(ipconfig getifaddr en0)
```
Comment (SchwarzM) [inline docs/guide/common_addons.md]: is this working? i thought a tag named 10.46.0.1:5000 in docker is not the same as localhost:5000 is not the same as 127.0.0.1:5000

File: docs/guide/common_addons.md (hunk 1 above)
Comment (jonathanschwarze) [reply docs/guide/common_addons.md]: True. I'd change it to 5g + add to the description that the value should be adjusted regarding the desired size and available quota.

File: docs/guide/common_addons.md (hunk 2 above)
Comment (jonathanschwarze) [reply docs/guide/common_addons.md]: The tunnel works fine whereas push/pull via. port-forwarding on Mac doesn't work as expected. Other user are facing the same issue. It seems like forwarded port is not being used by docker for Mac. `Error response from daemon: Get http://localhost:5000/v2/: dial tcp [::1]:5000: getsockopt: connection refused`. The annoying aspect by using the tunnel is to change the local_ip in "insecure repos" on the local machine after getting a new one.

**Pull request merged**